File: dynamic_listings/lua/default/collisiondetection/tile_conversion.lua
TILE_WIDTH = 32
TILE_HEIGHT = 32

local function convert_pixels_to_tile(x, y)
    -- Converts a point into tile coordinates
    local tile_x = math.floor(x / TILE_WIDTH)
    local tile_y = math.floor(y / TILE_HEIGHT)
    return {tile_x, tile_y}
end

File: dynamic_listings/python/default/collisiondetection/tile_conversion.py
from math import floor
TILE_WIDTH: int = 32
TILE_HEIGHT: int = 32


def convert_pixels_to_tile(x: int, y: int) -> tuple[int, int]:
    """
    Converts a point into tile coordinates

    :x: The pixel X coordinate
    :y: The pixel Y coordinate
    :returns: A 2-tuple containing the tile coordinates detected

    """
    tile_x: int = floor(x / TILE_WIDTH)
    tile_y: int = floor(y / TILE_HEIGHT)
    return (tile_x, tile_y)
